views/pages/legacy.njk

```diff
@@ -0,0 +1,23 @@
+{% import "components/hero.njk" as hero %}
+{% import "components/headers.njk" as headers %}
+{% import "components/segment.njk" as segment %}
+
+{% extends "layouts/main.njk" %}
+
+{% block title %}{{ title }} | {% endblock %}
+
+{% block content %}
+
+    {% set heroContent %}
+        {{ headers.overlayText('h2', title, 't2') }}
+    {% endset %}
+
+    {{ hero.header('foi', heroContent, 'turquoise') }}
+
+    <article role="main" class="nudge-up s-prose">
```

Comment:
Try adding `s-prose--full` too as that adds some more details typographic tweaks (in retrospect perhaps these classes should be merged)

Reply:
Good call, the template here is pretty much a proof-of-concept.